cmd/create/cluster/cmd.go

```diff
@@ -508,6 +511,14 @@ func init() {
 			"roles in STS clusters.",
 	)
 
+	// Options releated to HyperShift:
+	flags.BoolVar(
+		&args.hostedClusterEnabled,
+		"hosted-cp",
+		false,
+		"Enable or disable use of Hosted Cluster mode.",
```

Comment:
I think this should reflect better what the flag does from the customer point of view:

Maybe something like `"Enable the use of hosted control planes (HyperShift)."`